runtime/development/src/lib.rs

```diff
@@ -1890,6 +1903,7 @@ construct_runtime!(
 		ForeignInvestments: pallet_foreign_investments::{Pallet, Storage, Event<T>} = 117,
 		OraclePriceFeed: pallet_oracle_feed::{Pallet, Call, Storage, Event<T>} = 118,
 		OraclePriceCollection: pallet_oracle_collection::{Pallet, Call, Storage, Event<T>} = 119,
+		Remarks: pallet_remarks::{Pallet, Call, Event<T>} = 130,
```

Comment:
Note that we have to jump to 130 now.

Reply:
I am running into the same issue in #1633 and propose append at the end of all indices instead of taking the rather slim `130 to 149` range. This way, we only introduce one jump. For the `130 to 149` range I fear that we occupy all indices by the end of the year.

I propose an arbitrary 250 start index for the second batch of internal pallets: https://github.com/centrifuge/centrifuge-chain/blob/08f80ce49df346d8796b8a88e7c84332185f1654/runtime/development/src/lib.rs#L1950-L1951

Reply:
OK, will change that now. Thank you!